arangod/Cluster/ClusterInfo.cpp

```diff
@@ -3346,7 +3350,8 @@ Result ClusterInfo::createCollectionsCoordinator(
 
     if (info.state == ClusterCollectionCreationState::DONE) {
       // This is possible in Enterprise / Smart Collection situation
-      (*nrDone)++;
+      report->doUnderLock(
+          [](CreateCollectionReport& report) { report.nrDone++; });
```

Comment:
It was atomic, why do you decide to change it to mutex?

Reply:
I don't think here exist difference in terms of simplicity

Reply:
My thought here was mainly to keep everything that was updated under the same lock before also update und the same lock, therefore I placed all atomics into one guarded structure.
I agree that is potentially less performant than before, but we are waiting on all-server round-trips here, so mutex performant should not fall into account at all.